addons/common/functions/fnc_unloadPersonLocal.sqf

```diff
@@ -66,15 +70,15 @@ if (local _vehicle) then {
 
 [{
     params ["_unit", "_emptyPos"];
-    (alive _unit) && {(vehicle _unit) != _unit}
+    (alive _unit) && {(objectParent _unit) != _unit}
```

Comment:
```suggestion
    (alive _unit) && {!isNull objectParent _unit}
```

I don't really understand what this block is supposed to do, though: alive and in a vehicle as condition?

Reply:
I think it was designed to wait until the unit actually got out of the vehicle and then move them to the empty pos
but the logic was inverted and it's never done anything